README.md

```diff
@@ -22,3 +22,25 @@ Tested on Ubuntu 18.04 and Debian 10.
 * Put all these files in `/var/www/html`
 * Run `sudo setup.sh`
 * Visit http://localhost in your browser
+
+FAQ
+----
+**Can you delete it when you are done with it?**
+
+Yes. For any wiki you create, you will see a `Delete` link in the `Action` column of this table: https://patchdemo.wmflabs.org/
+
+**How long do the Patch demo wiki instances last for?**
+
+There is no definitive time after which wikis will automatically be deleted. With this said, we make no guarantees about how long they will continue to exist. A Patch demo wiki you've created could be deleted if we need to free up disk space to create space for new ones.
+
+**Can Patch demo wikis be named?**
+
+Wikis can not been named *within* Patch demo. Patch demo wikis currently inherit the name of the patch on which they are built.
```

Comment:
It depends what you mean by "name". There is no "name" label in the table. Wikis are listed by creator and the list of patches (potentially multiple) used to create it. They are also assigned a random hash, which becomes part of the URL. If you are referring to $wgSitename, which is used to set the page title and other labels on the wiki itself, then that can not currently be set, although it may be settable in the future.

Reply:
(I don't want to allow setting names because I don't want people to start relying on a particular wiki existing for a long time.)

Reply:
Sounds good. I've updated the PR to include both points above.